src/app/organization/loan-provisioning-criteria/view-loan-provisioning-criteria/view-loan-provisioning-criteria.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container">
```

Comment:
@BLasan The UI needs some major changes to make it consistent with other modules. I have updated the UI, please get the code from the [link](https://gist.github.com/muskankhedia/be63d448e7d6ddbbec97b3c5ef7ed5d2)  and update the UI. Also, add a screenshot of the updated UI in the description.